### PR TITLE
Add README.md text to help others host index meta-databases

### PR DESCRIPTION
**Pull request by rartino**
This PR adds a segment of text to README.md that explains how one can easily fork this repository to host one's own index meta-database that fulfills the requirements to be added to OPTiMaDe's central list of providers.

A preview is visible at: https://deploy-preview-18--optimade-providers.netlify.com/

### CODE REVIEW

File: README.md
Comment (CasperWA) [inline README.md]:
```suggestion
- Edit `/src/info/v1/links.json` to point out your OPTiMaDe databases:
```

File: README.md
Comment (CasperWA) [inline README.md]: This is new!
I like it - but it defeats our idea of "supporting" only two levels (as we write in the spec [here](https://github.com/Materials-Consortia/OPTiMaDe/blob/develop/optimade.rst#parent-and-child-objects)) - whatever support means in this context.
I thought the way to include the providers, was simply to include them with the type `provider`, again, as is indicated in the spec [here](https://github.com/Materials-Consortia/OPTiMaDe/blob/develop/optimade.rst#provider-objects).
However! I am all for not replicating too much, i.e., having the same providers spread out over all the different implementations. And having the option to direct the client to a continuously updated list of providers seems to be a good one to me.

My only issue with this then, is actually how we should present this link to the list of providers. While this may be one way, it is a bit misinformative - since the list of providers is not a parent of the index meta-database, but rather it's on the same level (in some sense).
Another option would be to introduce a fourth type, like the plural of provider, i.e., `"providers"` or a combined type of `"provider_list"`.
Another option would be to introduce a type that refers to implementation on the same "level", like `"sibling"` (to follow on with the `child` and `parent` nomenclature), but that feels a bit mis-informative as well, I feel.

So in the end, I am open to ideas, and it may even end up with what you have here. I would just like to raise the discussion to make sure it all makes sense with the rest of the spec (or adapt the spec accordingly).

File: README.md
Comment (CasperWA) [inline README.md]: I don't know about these commands, but I assume you have written them correctly :)

File: README.md
Comment (CasperWA) [inline README.md]:
```suggestion
  - You are also recommended to set your subdomain in *Domain settings*, or setup your own custom domain.
```

File: README.md
Comment (rartino) [reply README.md]: Oh... I had somehow gotten the idea that this was how it was supposed to work so that an unbroken tree was presented to the client, which could be traversed both up and down. But you are right that the present specification certainly do not mandate this setup. In particular, the fact that providers are linked out with `provider`-type objects instead of `child` already breaks this symmetry.

So, for now I should just remove this segment. But I like your version with a `provider_list` link.

Comment (rartino) [reply README.md]: However, maybe my confusion ties into #217 a bit. Should these link objects really have these kinds of different types, or would it be better to change them to be `links`-type and put the relationship as an attribute field?

File: README.md
Comment (rartino) [reply README.md]: I did test the procedure as I stated to prepare for the omdb index meta-database at https://github.com/rartino/providers/tree/omdb ; https://omdb.netlify.com

File: README.md
Comment (rartino) [reply README.md]: It is great to get things like these carefully proofed! Thanks!

**Comment by rartino** [comment]
I've updated this to confirm with the latest changes and utilize PR #34.

**Comment by giovannipizzi** [comment]
@rartino After #34, do we still want to have people create a new repo to have a meta-DB? (maybe yes if they want to host it still themselves to decide how often to make changes?)

I would then wait for #34 to be merged, clarify when one should create a new repo vs. just adding the files here, and check again the docs you wrote (I had to do some renaming in #34 after you updated this PR)

**Comment by rartino** [comment]
> @rartino After #34, do we still want to have people create a new repo to have a meta-DB? (maybe yes if they want to host it still themselves to decide how often to make changes?)

@giovannipizzi Yes, this is still relevant. I'm going to use this setup to keep an index database that remains under my control.

You can look at it as this, there is a range of user needs:
1. "I will never provide anything but a single url".
2. "I will do changes now and then".
3. "Our system dynamically generated databases on the fly" 

Users category 1 hosts their index meta-database in the providers repo. Cloning the repo is for users in category 2.

**Comment by giovannipizzi** [comment]
Ok I see, fine! I'll wait for #34 to be merged first before reviewing, as there might be further changes requested that might affect this PR

**Review by CasperWA — APPROVED** [review]
I think we should just get this in by now. It's good information.

**Review by giovannipizzi — APPROVED** [review]

**Comment by giovannipizzi** [comment]
There're some conflict that need to be solved before merging

**Comment by CasperWA** [comment]
@rartino I will resolve the conflicts here and merge this.

**Pull request merged**
